### PR TITLE
Added Issue and PR Templates

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <aadithya794@gmail.com>

This PR adds Issue & PR Templates to the repository and also updated the README.md file

r? @Licenser

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+NOTE: the total number is irrelevant as it will vary from system to system. The delta is what
+matters.
+
+If the benchmarks fail on your system, note it in the issue, and someone will try to run them for
+you.
+
+If your changes do not affect performance, and you can argue this, do it in this section, it is a
+valid response.
+
+-->
```
Comment (Licenser) [inline .github/pull_request_template.md:61]: I think a lot of this isn't relevant for the webpage, the RFC process isn't important, there are no user-facing changes that would go to the docs, unsafe isn't a thing, performance impact here while not nonexistent is not as big of a topic as with the main application and won't warrant a section here.

Tested code could be  relevant if the web page ever gets code included

**Review by Licenser — CHANGES_REQUESTED** [review]
Thank you :) this is nice just a few changes to the PR template would be nice as the one from tremor-rs doesn't really make sense without adoption

**Comment by adithyaakrishna** [comment]
Thanks a lot @Licenser for reviewing this, I'll make the requested changes and Push it soon :)

File: .github/pull_request_template.md (hunk 1 above)
Comment (adithyaakrishna) [reply .github/pull_request_template.md:61]: Yeah, after a close review this makes sense to me too. Will change it Soon 🚀

**Comment by Licenser** [comment]
closing this since we're depricating this repository

**Pull request closed without merging**